can-define.js

```diff
@@ -719,6 +719,14 @@ define.setup = function(props, sealed) {
 };
 define.replaceWith = replaceWith;
 define.eventsProto = eventsProto;
+define.addBaseEvents = function(objPrototype){
+	objPrototype._baseEvents = objPrototype.addEventListener &&
```

Comment:
_baseEvents is going to be shared by every instance of `objPrototype`.  As `can-event` mutates `this` (it keeps event listeners on `this`) ... a single `_baseEvents` could be a problem.